+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Wed Sep 27 19:35:45 2023 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL iSulad
+ pkgs=iSulad
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs iSulad --node 1 --tempfile ''
Wed Sep 27 19:35:56 2023 - INFO  - pkgs:(iSulad) is already installed
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ cp /etc/isulad/daemon.json /etc/isulad/daemon.json.bak
cp: cannot stat '/etc/isulad/daemon.json': No such file or directory
+ sed -i '/registry-mirrors/a\"https:\/\/ariq8blp.mirror.aliyuncs.com\"' /etc/isulad/daemon.json
sed: can't read /etc/isulad/daemon.json: No such file or directory
+ systemctl restart isulad.service
Job for isulad.service failed because the control process exited with error code.
See "systemctl status isulad.service" and "journalctl -xeu isulad.service" for details.
+ isula pull busybox
Image "busybox" pulling
Can not connect with server.Is the iSulad daemon running on the host?
+ isula create -it busybox
Can not connect with server.Is the iSulad daemon running on the host?
++ isula ps -a
++ grep -vi NAMES
++ awk '{print $NF}'
Can not connect with server.Is the iSulad daemon running on the host?
+ container_id=
+ isula start ''
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Wed Sep 27 19:35:59 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Wed Sep 27 19:36:00 2023 - INFO  - Start to run test.
+ test -d testdir
+ mkdir testdir
+ touch testdir/mytest.txt
+ isula cp testdir :/home
must specify at least one container source
+ CHECK_RESULT 1 0 0 'File copy failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='File copy failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'File copy failed'
+ LOG_ERROR 'File copy failed'
+ message='File copy failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'File copy failed'
Wed Sep 27 19:36:01 2023 - ERROR - File copy failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_container02.sh line 40'
+ message='oe_test_iSulad_container02.sh line 40'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_container02.sh line 40'
Wed Sep 27 19:36:01 2023 - ERROR - oe_test_iSulad_container02.sh line 40
+ '[' 0 -eq 1 ']'
+ return 0
+ isula exec -it '' /bin/sh -c 'ls /home/testdir'
+ CHECK_RESULT 1 0 0 'There is no such file or directory'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='There is no such file or directory'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'There is no such file or directory'
+ LOG_ERROR 'There is no such file or directory'
+ message='There is no such file or directory'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'There is no such file or directory'
Wed Sep 27 19:36:02 2023 - ERROR - There is no such file or directory
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_container02.sh line 42'
+ message='oe_test_iSulad_container02.sh line 42'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_container02.sh line 42'
Wed Sep 27 19:36:03 2023 - ERROR - oe_test_iSulad_container02.sh line 42
+ '[' 0 -eq 1 ']'
+ return 0
+ isula cp :/home /tmp
must specify at least one container source
+ CHECK_RESULT 1 0 0 'File copy failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='File copy failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'File copy failed'
+ LOG_ERROR 'File copy failed'
+ message='File copy failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'File copy failed'
Wed Sep 27 19:36:04 2023 - ERROR - File copy failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_container02.sh line 44'
+ message='oe_test_iSulad_container02.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_container02.sh line 44'
Wed Sep 27 19:36:04 2023 - ERROR - oe_test_iSulad_container02.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ test -f /tmp/home/testdir/mytest.txt
+ CHECK_RESULT 1 0 0 'There is no such file'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='There is no such file'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'There is no such file'
+ LOG_ERROR 'There is no such file'
+ message='There is no such file'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'There is no such file'
Wed Sep 27 19:36:05 2023 - ERROR - There is no such file
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_container02.sh line 46'
+ message='oe_test_iSulad_container02.sh line 46'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_container02.sh line 46'
Wed Sep 27 19:36:05 2023 - ERROR - oe_test_iSulad_container02.sh line 46
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Wed Sep 27 19:36:06 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 4
+ test 4 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Wed Sep 27 19:36:07 2023 - ERROR - failed to execute the case.
+ exit 4
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Wed Sep 27 19:36:07 2023 - INFO  - Start to restore the test environment.
+ isula ps -a
+ grep -vi NAMES
+ xargs -I '{}' isula stop '{}'
+ awk '{print $NF}'
Can not connect with server.Is the iSulad daemon running on the host?
+ isula ps -a
+ grep -vi NAMES
+ xargs -I '{}' isula rm -f '{}'
+ awk '{print $NF}'
Can not connect with server.Is the iSulad daemon running on the host?
+ isula rmi busybox
Can not connect with server.Is the iSulad daemon running on the host?
+ mv -f /etc/isulad/daemon.json.bak /etc/isulad/daemon.json
mv: cannot stat '/etc/isulad/daemon.json.bak': No such file or directory
+ rm -rf testdir /tmp/home
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Wed Sep 27 19:36:09 2023 - WARN  - no thing to do.
+ return 0
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Wed Sep 27 19:36:09 2023 - INFO  - End to restore the test environment.
